services/backend/src/config.js

```diff
@@ -69,7 +69,7 @@ const development = {
     "/auth/realms/individual/protocol/openid-connect/token",
   GEDSAPIURL:
     GEDSAPIURL ||
-    "https://geds-sage-ssc-spc-apicast-production.api.canada.ca/gapi/v2/",
+    "https://geds-ssc-spc-apicast-production.api.canada.ca/gapi/v2/",
```

Comment:
why has this changed? Im just curious

Reply:
this should be changed back, the link with `sage` is the correct one